ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/Types.hs

```diff
@@ -1070,6 +1579,9 @@ data TracePeerSelection peeraddr =
 
      | TraceChurnWait          DiffTime
      | TraceChurnMode          ChurnMode
+     | TraceChurnAction  DiffTime ChurnAction Int
+     | TraceChurnTimeout DiffTime ChurnAction Int
```

Comment:
Maybe we can add a short haddock that the last parameters is the change in number of peers after a particular churn action finishes

Reply:
Done; I added haddocks to all three fields.